plugins/gtav/structs.h

```diff
@@ -0,0 +1,227 @@
+// Copyright 2022 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#ifndef GTAV_STRUCTS
+#define GTAV_STRUCTS
+
+#include <cstdint>
+
+// Reference: https://github.com/Yimura/GTAV-Classes
+
+typedef uint64_t ptr_t;
+
+enum class GameState : int32_t {
+	Invalid = -1,
+	Playing,
+	Died,
+	Arrested,
+	FailedMission,
+	LeftGame,
+	Respawn,
+	InMPCutscene
+};
+
+enum class ModelType : uint8_t { Unk0, Object, Unk1, Unk2, Weapon, Vehicle, Ped, Plant = 129 };
+
+namespace rage {
+template< typename T > union vector3 {
+	T data[3];
+	struct {
+		T x, y, z;
+	};
+};
+
+typedef vector3< float > fvector3;
+
+struct netAddress {
```

Comment:
Class names should start with a capital letter (also applies to class names below)

Reply:
The name is from the game though.

Reply:
Meh... Alright then though

Reply:
I mean, we can use any names we want, but I would like to adhere to the original ones for better reference.